src/content/learn/managing-state.md

```diff
@@ -22,7 +22,7 @@ Povećavanjem aplikacije, važno je da budete svesni kako vam je state organizov
 
 ## Reagovanje na input pomoću stanja {/*reacting-to-input-with-state*/}
 
-Sa React-om, nećete direktno u kodu menjati UI. Na primer, nećete pisati komande poput "onemogući dugme", "omogući dugme", "prikaži uspešnu poruku", itd. Umesto toga, opisaćete kakav UI želite da vidite za različita vizuelna stanja vaše komponente ("inicijalno stanje", "stanje pisanja", "uspešno stanje"), a onda ćete pokrenuti promene state-a kao odgovor na korisnički input. Ovo je slično onome kako dizajneri razmišljaju o UI-u.
+Sa React-om, nećete direktno u kodu menjati UI. Na primer, nećete pisati komande poput "onemogući dugme", "omogući dugme", "prikaži uspešnu poruku", itd. Umesto toga, opisaćete kakav UI želite da vidite za različita vizuelna stanja vaše komponente ("inicijalno stanje", "stanje pisanja", "uspešno stanje"), a onda ćete pokrenuti promene stanja kao odgovor na korisnički input. Ovo je slično onome kako dizajneri razmišljaju o UI-u.
```

Comment:
Please pay attention on places where I used terms `stanje` and `state`. Since it's the same word on english I needed to distinguish React's state and state from state machine (`automat` on Serbian).